doc/History.md

```diff
@@ -43,8 +44,9 @@ So far the list includes:
 * MongooseICE
 
 
-### Soon: IoT and chabots
+### 2018-2019: Planetary architecture, to welcome IoT-scale and chabots
 
-The next step on our journey with the MongooseIM platform is delivering a full stack IoT solution.
+The next step on our journey with the MongooseIM platform is delivering a planetary architecture, so that we can welcome the massive amounts of "users" that come with a full stack IoT and chatbot solution.
```

Comment:
The next step on our journey with the MongooseIM platform is to enable building planetary scale architectures.
This is necessary to welcome the massive influx of users that come with a full stack IoT and chatbot solutions.
The ability to connect robots and humans is the requirement of the next technological breakthrough.